trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk

```diff
@@ -121,8 +121,6 @@
 {% set keyingSheetHtml %}
   <h2 class="govuk-heading-l" data-cy="keying-sheet-heading">Keying sheet</h2>
 
-  <p class="govuk-body">Select payments and mark as done when the adjustments have been keyed into ACBS.</p>
```

Comment:
I think we only wanted to delete this for PDC_READ users not just everyone, so we want to wrap this in an:
`{% if userCanEdit %}`

Could you open a new PR to add this please?

Reply:
will readd in PR for 3451